great_expectations/core/util.py

```diff
@@ -69,23 +69,37 @@
 
 
 def nested_update(
-    d: Union[Iterable, dict], u: Union[Iterable, dict], dedup: bool = False
+    d: Union[Iterable, dict],
+    u: Union[Iterable, dict],
+    dedup: bool = False,
+    concat_lists: bool = True,
```

Comment:
Could you provide some context as to why this changed? Does one of the above functions invoke it?

Also, I didn't write a test for this method but we might want to add a unit test that sets `concat_lists` to `False`? I wasn't entirely sure so I left it as is.